network-core/src/server/transaction.rs

```diff
@@ -3,6 +3,7 @@
 use crate::codes;
 use crate::error::Code as ErrorCode;
 
+use chain_core::property;
```

Comment:
I tend to use `self` inside the brackets, but it's a matter of taste really.

Reply:
And we could just use the `property::` path consistently in this module, for clarity.